src/main/java/io/gravitee/repository/jdbc/management/JdbcApiKeyRepository.java

```diff
@@ -139,7 +142,7 @@ private boolean addClause(boolean first, StringBuilder query) {
     public Set<ApiKey> findByPlan(String plan) throws TechnicalException {
         LOGGER.debug("JdbcApiKeyRepository.findByPlan({})", plan);
         try {
-            List<ApiKey> items = jdbcTemplate.query("select * from " + escapeReservedWord("keys") + " where plan = ?"
+            List<ApiKey> items = jdbcTemplate.query("select * from " + escapeReservedWord(this.tableName) + " where plan = ?"

```

Comment:
escape the table name in the constructor
```suggestion
            List<ApiKey> items = jdbcTemplate.query(getOrm().getSelectAllSql() + " where plan = ?"

```